### ISSUE TITLE
Bug: we should show "uploading" when sending attachment

### ISSUE THREAD

**Issue by compulim** [Microsoft/BotFramework-WebChat#1265]
> This is on-par as of v3.

![image](https://user-images.githubusercontent.com/1622400/47759248-ecf6aa80-dc6b-11e8-9a79-d19629d48a84.png)

Instead of timing out thru the regular 5 seconds rule, we should show "uploading" and allow more time to complete.

**Comment by corinagum** [comment]
@mewa1024  - placeholder for Marc

**Comment by DesignPolice** [comment]
Yes I think we should just use a loader like this one, CSS would be easier than a gif.  


![screen shot 2018-12-06 at 3 54 25 pm](https://user-images.githubusercontent.com/45223620/49618918-4179fd80-f96f-11e8-99ee-c4895b653fbf.png)

**Comment by corinagum** [comment]
When this is implemented, [Offline UI](https://github.com/Microsoft/BotFramework-WebChat/issues/1523) will have a spinner usable in this issue. Implementer should use that to prevent double-effort.

**Comment by DesignPolice** [comment]
I went through this again, and captured a visual of some of a couple of options we could look at, and what is going on Teams and Skype for reference. Hopefully, this will help with determining a solution and the time it will take to implement. Thanks 

![#1265v2](https://user-images.githubusercontent.com/45223620/55639093-7a8b7880-577d-11e9-80de-3d57bf5987a3.png)

**Comment by corinagum** [comment]
Thanks @DesignPolice! Will mention this at standup today. I know you like the Skype icons on the right but I still prefer the Teams example on the left. :P

Web Chat doesn't yet have input text wrap (top right spec), so that version would need to wait for #17 to be fully implemented. Is it okay to not have word wrap immediately? 

A few comments:
-It seems redundant to have text above the attachments when there is also a blank send box below. 
- Because there are different colors depending on file type, we will also need specs on what colors = what file types, icons, etc. It looks like Office documents can use their own icon, and for images the icon is the paperclip, but what about other files? I think the paperclip was originally swapped out for the old image icon to show that pretty much any file type may be uploaded, as opposed to indicating just images.

I am also concerned that this is more like an XL t-shirt size for me to implement, and might take longer than the time we have until 4.4 (alongside other issues). Thoughts?

**Comment by DesignPolice** [comment]
Sounds good @corinagum. 

I can live with the Skype version as well, it is much cleaner - I don't know what is involved in adding the cropped images - how much code from Skype or Teams we can emulate.

Yes, I can spec colors and icons if we go that direction as well... I think core MSFT apps (Word, Excel, PPT, OneNote if those are sent?) PDF, gif? video? and maybe just a clip for anything else?  I can be flexible on that based on the "T-Shirt" size that we can afford or fit in? (to continue the odd t-shirt analogy - are there shirts that don't fit??) 

If we needed to just do a clip and a "gray box" and name for everything because it saves time and fixes an issue we could look at that as well. Probably need to Chris or Scot to call the ball on that I know there are lots of needs.

You are correct on the message box, it should just be on top of the attachments - forgot to delete it was trying to go quickly.

**Comment by corinagum** [comment]
Tagging Chris for review

**Comment by cwhitten** [comment]
We're going to punt on this given the updated UX provided by @DesignPolice

**Comment by leolorenzoluis** [comment]
@compulim  I want to add a suggestion, please let us set the default timeout in the config. I don't see any reason why the devs can't override the timeout themselves.
